File: v2/java/src/main/java/io/swagger/client/model/PostRequest.java
package io.swagger.client.model;

import io.swagger.client.model.PostInput;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class PostRequest  {
  
  private PostInput post = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("post")
  public PostInput getPost() {
    return post;
  }
  public void setPost(PostInput post) {
    this.post = post;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class PostRequest {\n");
    
    sb.append("  post: ").append(post).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/PostRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Postrequest {
    

    
    public PostInput Post { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Postrequest {\n");
      
      sb.Append("  Post: ").Append(Post).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}